portal-impl/test/unit/com/liferay/portal/json/JSONUtilTest.java

```diff
@@ -220,15 +223,18 @@ public void testIsArray() {
 	}
 
 	@Test
-	public void testisValid() {
+	public void testIsJSONObject() {
 		Assert.assertTrue(
 			"null is an empty JSON string", JSONUtil.isJSONObject(null));
 
-		Assert.assertTrue("\"\" is an empty JSON string", JSONUtil.isJSONObject(""));
+		Assert.assertTrue(
+			"\"\" is an empty JSON string", JSONUtil.isJSONObject(""));
```

Comment:
@alecsloan we should use `StringPool.BLANK` here.
```suggestion
			"\"\" is an empty JSON string", JSONUtil.isJSONObject(StringPool.BLANK));
```